striatum/bandit/bandit.py

```diff
@@ -58,35 +75,48 @@ def actions(self):
         return self._actions
 
     @abstractmethod
-    def get_action(self, context):
+    def get_action(self, context, n_action=1):
         """Return the action to perform
 
-        Parameters
-        ----------
-        context : {array-like, None}
-            The context of current state, None if no context avaliable.
+            Parameters
+            ----------
+            context : dictionary
+                Contexts {action_id: context} of different actions.
```

Comment:
No. We are following a convention, and he is changing the convention without reason. 
(That's not good to have so many deep indent.)